spec/models/application_record_spec.rb

```diff
@@ -67,6 +67,21 @@
     end
   end
 
+  describe 'data_rows' do
+    let :call do
+      @record.data_rows
+    end
+    before :each do
+      @record = create :application_record,
+                       data: [['a question', 'an answer']]
```

Comment:
So here's a thought--should these tasks and methods be deleted, or merely updated?